blockchain/utxocache.go

```diff
@@ -617,6 +617,7 @@ func (b *BlockChain) InitConsistentState(tip *blockNode, interrupt <-chan struct
 
 		// Set the last flush hash as it's the default value of 0s.
 		s.lastFlushHash = tip.hash
+		s.lastFlushTime = time.Now()
```

Comment:
Independent bug fix? 

Or before time would be the zero value of `time.Time` so the subsequent checks work properly as any time is after `timeZero`?

Reply:
Yeah this is a separate fix. The only thing it affects is the immediate flushing on start when upgrading from a v0.23 node.

I can make it into a separate PR if requested.